common/app/views/fragments/basher.scala.html

```diff
@@ -0,0 +1,23 @@
+@(id: String)(implicit request: RequestHeader)
+
+@import views.support.Basher
+
+@Basher.fromId(id).map { id =>
```

Comment:
This overrides the `id` identifier which hurts readability. `fromId` returns a `basherType`, so maybe name it that?